invenio_vocabularies/alembic/676dd587542d_create_funders_vocabulary_table.py

```diff
@@ -0,0 +1,67 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Create funders vocabulary table."""
+
+import sqlalchemy as sa
+import sqlalchemy_utils
+from alembic import op
+from sqlalchemy.dialects import mysql, postgresql
+
+# revision identifiers, used by Alembic.
+revision = '676dd587542d'
+down_revision = '17c703ce1eb7'
+branch_labels = ()
+depends_on = None
+
+
+def upgrade():
+    """Upgrade database."""
+    op.create_table(
+        'funder_metadata',
+        sa.Column(
+            'created',
+            sa.DateTime().with_variant(mysql.DATETIME(fsp=6), 'mysql'),
+            nullable=False
+        ),
+        sa.Column(
+            'updated',
+            sa.DateTime().with_variant(mysql.DATETIME(fsp=6), 'mysql'),
+            nullable=False
+        ),
+        sa.Column(
+            'id',
+            sqlalchemy_utils.types.uuid.UUIDType(),
```

Comment:
This should have a `length=16`, same as teh `ChoiceType` below a `length=1`. However, it is not compatible with alembic :/

Reply:
I see that in the `names` recipe it's the same. Isn't it fine to leave it like this?

Reply:
yes, it is. I just added a comment in case of doubt.